config/deploy/production.rb

```diff
@@ -1,7 +1,5 @@
 # frozen_string_literal: true
-remote_user = Net::SSH::Config.for('af-transit-app3.admin.umass.edu')[:user] || ENV['USER']
```

Comment:
Looks great, I just wonder why we don't have to do this any more?

Reply:
Some upgrade of `Net::SSH` or `sshkit`, I assume?  It's pretty obvious default behavior, IMO

It's also one of those things that is redundant but doesn't break anything, so if you're constantly copy-and-pasting configs when you make a new project, who knows how long this hasn't been necessary?

I did confirm, however, that changing my `User` in my ssh config still changed the login Capistrano used.